app/app/locations/[location_id]/page.tsx

```diff
@@ -60,6 +87,21 @@ export default function LocationDetailsPage({
           />
         </Col>
       </Row>
+      <Row>
+        <Col>
+          <Card>
+            <Card.Header>Crashes</Card.Header>
+            <Card.Body>
+              <TableWrapper
+                columns={locationCrashesColumns}
+                initialQueryConfig={locationCrashesQueryConfig}
+                localStorageKey="locationCrashesQueryConfig"
+                contextFilters={locationIdFilter}
```

Comment:
new `contextFilters` prop that let's us pass in this filter from local storage. The main challenge here is that the primary `queryConfig` gets saved in local storage, and we don't to include the `location_id` in the local storage config, because it's a dynamic value.

I thought about a few a different ways to approach, but this seemed simple enough for now. I believe this is the only place in the app where we'll need to use this new feature.

Reply:
can you mention in your comment the need for the `location_id` filter to be dynamic unlike the query config thats saved in local storage? i feel like reading your comment here was super helpful but i might have been confused if i just was looking at the code